test/core/client_channel/resolvers/binder_resolver_test.cc

```diff
@@ -23,6 +23,7 @@
 
 #include <cstring>
 
+#include <grpc/grpc.h>
```

Comment:
why does this need to be included now?

Reply:
grpc_init gets called - it used to be accidentally included from src/core/lib/channel/channel_args.h, but since that got cleaned up other things need to also.